### PR TITLE
Re-enable rebuild indexing plan debouncer

### PR DESCRIPTION
**Pull request by guilload**
Re-enabled debouncer and fixed ingest v2 happy path test. `test_single_node_cluster` needs more work, I'll open a separate PR.

### CODE REVIEW

File: quickwit/quickwit-control-plane/src/control_plane.rs [hunk 1]
```diff
@@ -66,9 +66,9 @@ use crate::IndexerPool;
 
 /// Interval between two controls (or checks) of the desired plan VS running plan.
 pub(crate) const CONTROL_PLAN_LOOP_INTERVAL: Duration = if cfg!(any(test, feature = "testsuite")) {
-    Duration::from_millis(500)
+    Duration::from_millis(100)
```
Comment (guilload) [inline quickwit/quickwit-control-plane/src/control_plane.rs:69]: Tests are slow to run.

File: quickwit/quickwit-control-plane/src/control_plane.rs [hunk 2]
```diff
@@ -66,9 +66,9 @@ use crate::IndexerPool;
...
 } else {
-    Duration::from_secs(3)
+    Duration::from_secs(5)
```
Comment (guilload) [inline quickwit/quickwit-control-plane/src/control_plane.rs:71]: We now react to nodes joining and leaving the cluster, so I think it makes sense to improve that value now.

File: quickwit/rustfmt.toml [hunk 3]
```diff
@@ -6,6 +6,6 @@ comment_width = 120
 format_strings = true
 group_imports = "StdExternalCrate"
 imports_granularity = "Module"
-normalize_comments = false
+normalize_comments = true
```
Comment (fulmicoton) [inline quickwit/rustfmt.toml:9]: I think this is a bad idea. `/* */` are very useful when commenting out code blocks.

**Review by fulmicoton — APPROVED** [review]
I think we should cook some solution to return a one shot receiver, to allow clients to wait for the next build plan to be executed.

File: quickwit/rustfmt.toml (hunk 3 above)
Comment (guilload) [inline quickwit/rustfmt.toml:9]:
```suggestion
normalize_comments = false
```

Comment (guilload) [reply quickwit/rustfmt.toml:9]: I just reverted your previous commit, but I can put it back to `false`.

**Pull request merged**